Gemfile

```diff
@@ -6,6 +6,7 @@ gemspec
 group :test do
   gem 'rails', detect_rails_version
   gem 'activeadmin', github: 'activeadmin', ref: 'ba285ea2fc23ade108ed582134cae8c641f910a4'
```

Comment:
Could you bump activeadmin version to reproduce the issue?

Reply:
I can do it when this is merged - https://github.com/activeadmin/activeadmin/pull/5253.

Reply:
Obnovil